src/style.css

```diff
@@ -1 +1,89 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
```

Comment:
It's recommended to provide a fallback font in case the custom font fails to load. Consider adding a generic font family as a fallback.